functions/calculate-score/function.json

```diff
@@ -0,0 +1,14 @@
+{
+  "disabled": false,
+  "bindings": [
+    {
+      "schedule": "0 0 21 * * *",
```

Comment:
9pm?  i know these are not valid CRON expressions, hence why i ask...

Reply:
@GuyHarwood this serves as a placeholder since there is no direct requirement with regards to the time of the day where this function will be executed.
It can be decided before merging.

Reply:
ok, but what does the expression mean?

Reply:
Means that it will execute `every day at 21:00:00`